src/create-type.test.js

```diff
@@ -22,6 +22,14 @@ describe('KISS type', () => {
       __.assertThat(() => type.get.unknown, failsNicely)
       __.assertThat(() => type.set.unknown, failsNicely)
     })
+    it('type properties can be console logged', () => {
+      const doesNotThrowError = __.not(
+        __.throws(
+          __.typedError(TypeError, __.containsString('Cannot convert a Symbol value to a string'))
+        )
+      )
+      __.assertThat(() => console.log(type.props), doesNotThrowError)
+    })
```

Comment:
What do you think about just `console.log(type.props)` as the assertion (in the sense that there are not runtime errors)?

Reply:
im a fan